Clinical-Genomics/scout
# Edit/create panels with genes in build 38 (hotfix)

This is a branch based on branch [version_4.36](https://github.com/Clinical-Genomics/scout/tree/version_4.36), so we can easily create an hotfix version including only this change. 

**OBS --> DO NOT MERGE INTO MASTER!!! --**

Fix #2726 

**How to test**:
1. On a local instance of Scout containing all genes (37 and 38) edit the demo gene panel with gene NOTCH2NLC (present only with genome build 38).
2. Try to load a new gene panel from file after modifying the file to include gene NOTCH2NLC. Tip: replace a gene from [this file](https://github.com/Clinical-Genomics/scout/blob/master/scout/demo/panel_1.csv) with  53924 and NOTCH2NLC.

In both cases the panel should be updated with the gene

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
